src/index_notation/index_notation.cpp

```diff
@@ -1510,25 +1522,43 @@ IndexStmt IndexStmt::divide(IndexVar i, IndexVar i1, IndexVar i2, size_t splitFa
   return transformed;
 }
 
-IndexStmt IndexStmt::precompute(IndexExpr expr, IndexVar i, IndexVar iw, TensorVar workspace) const {
+IndexStmt IndexStmt::precompute(IndexExpr expr, std::vector<IndexVar> i_vars,
+                                std::vector<IndexVar> iw_vars, TensorVar workspace) const {
+
+  // TODO: need to assert they are same length
   IndexStmt transformed = *this;
   string reason;
 
-  if (i != iw) {
-    IndexVarRel rel = IndexVarRel(new PrecomputeRelNode(i, iw));
-    transformed = Transformation(AddSuchThatPredicates({rel})).apply(transformed, &reason);
-    if (!transformed.defined()) {
-      taco_uerror << reason;
+ // FIXME: need to re-enable this later
```

Comment:
Is this fixme redundant? Not sure what needs to be re-enabled exactly.

Reply:
Yea the code below line 1532 was originally commented out and had to be added back in later. The FIXME is redundant.